### DISCUSSION TITLE
Terraform에서의 EIP 관리에 대해 질문있습니다.

### DISCUSSION THREAD

**Discussion by supernovachoi** [tedilabs/fastcampus-devops#88] [Q&A]
강사님 안녕하세요! Terraform 강의를 모두 듣고 난 후 

새 VPC를 구축 중입니다. 그러던 중 EIP 관리에대한 궁금증이 생겼는데 

미리 생성된 EIP와 연결해제된 EIP를 Pool 방식으로 재사용하려면 Terraform에서 어떻게 해야하나요?

**Comment by posquit0** [comment]
@supernovachoi 안녕하세요~ Pool 방식으로 재사용한다는 의미가 무엇인지 좀 더 구체적으로 설명해주시면 고민해보고 답변드리겠습니다! :)
